web-common/src/features/dashboards/dimension-table/dimension-table-export-utils.ts

```diff
@@ -0,0 +1,137 @@
+import { getQuerySortType } from "@rilldata/web-common/features/dashboards/leaderboard/leaderboard-utils";
+import { SortDirection } from "@rilldata/web-common/features/dashboards/proto-state/derived-types";
+import type { MetricsExplorerEntity } from "@rilldata/web-common/features/dashboards/stores/metrics-explorer-entity";
+import type { TimeControlState } from "@rilldata/web-common/features/dashboards/time-controls/time-control-store";
+import {
+  TimeComparisonOption,
+  TimeRangePreset,
+  TimeRangeType,
+} from "@rilldata/web-common/lib/time/types";
+import type {
+  V1MetricsViewComparisonRequest,
+  V1MetricsViewSpec,
+  V1TimeRange,
+} from "@rilldata/web-common/runtime-client";
+import { runtime } from "@rilldata/web-common/runtime-client/runtime-store";
+import { get } from "svelte/store";
+
+// Temporary. A future PR will add iso to the selection itself.
+const TIME_RANGES_TO_ISO: Record<TimeRangeType, string> = {
+  [TimeRangePreset.ALL_TIME]: "inf",
+  [TimeRangePreset.LAST_SIX_HOURS]: "PT6H",
+  [TimeRangePreset.LAST_24_HOURS]: "PT24H",
+  [TimeRangePreset.LAST_7_DAYS]: "P7D",
+  [TimeRangePreset.LAST_14_DAYS]: "P14D",
+  [TimeRangePreset.LAST_4_WEEKS]: "P4W",
+  [TimeRangePreset.LAST_12_MONTHS]: "P12M",
+  [TimeRangePreset.TODAY]: "rill-TD",
+  [TimeRangePreset.WEEK_TO_DATE]: "rill-WTD",
+  [TimeRangePreset.MONTH_TO_DATE]: "rill-MTD",
+  [TimeRangePreset.QUARTER_TO_DATE]: "rill-QTD",
+  [TimeRangePreset.YEAR_TO_DATE]: "rill-YTD",
```

Comment:
The `rill` prefix logic is reversed here, right? E.g. it should be `rill-PT6H` not `PT6H`, and `WTD` not `rill-WTD`?

Reply:
Not needed for Last-X. Since those easily map to the standard.